src/Chapter2.hs

```diff
@@ -336,7 +350,7 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList start end list = take (end - start + 1) (drop start list)
```

Comment:
The main idea is correct :+1: Though, it looks like this implementation doesn't work correctly on negative bounds. Corner cases sometimes can be tricky to discover and think about...

Reply:
Fixed